website/docs/plugins/optuna_sweeper.md

```diff
@@ -208,3 +208,46 @@ If `log` is `false`, the parameter is mapped to [`UniformDistribution`](https://
   - `choices`: a list of parameter value candidates
 
 The parameters are mapped to [`CategoricalDistribution`](https://optuna.readthedocs.io/en/stable/reference/generated/optuna.distributions.CategoricalDistribution.html).
+
+
+## Example 2:  Multi-Objective Optimization
+
+We include another example in the directory. `example/multi-objective.py` implements a simple benchmark function which has two objective values. We'd like to minimize both objectives simultaneously.
+
+You can discover the Optuna sweeper parameters with:
+
+```yaml title="python example/multi-objective.py hydra/sweeper=optuna --cfg hydra -p hydra.sweeper"
+# @package hydra.sweeper
+_target_: hydra_plugins.hydra_optuna_sweeper.optuna_sweeper.OptunaSweeper
+optuna_config:
+  direction:
+  - minimize
+  - minimize
+  storage: null
+  study_name: multi-objective
+  n_trials: 20
+  n_jobs: 1
+  sampler: nsgaii
+  seed: 123
+search_space:
+  x:
+    type: float
+    low: 0
+    high: 5
+    step: 0.5
+  y:
+    type: float
+    low: 0
+    high: 3
+    step: 0.5
+```
+
+To run optimization, run the following command in the `plugins/hydra_optuna_sweeper` directoy:
+
+```commandline
+python example/multi-objective.py --multirun
+```
+
+For this problem, no single solution exists that simultaneously minimizes both objectives. Instead, we obtained a set of solutions, namely Pareto optimal solutions, that shapes the trade-off relationship between the objectives. In the following figure, the blue dots show the Pareto optimal solutions in the optimization results.
```

Comment:
Is `Pareto optimal solutions` something common in multi-objective optimization? If not, we may want to add a small link somewhere?

Reply:
Thank you for pointing it out. In my understanding, it is a common term in the domain of multi-objective optimization. For example, it appears in the [Wikipedia article](https://en.wikipedia.org/wiki/Multi-objective_optimization).

On the other hand, it is not as common as the word `the best solution`, and I should add a link as you suggested. Do you think the link to the [Wikipedia article](https://en.wikipedia.org/wiki/Multi-objective_optimization)?

In addition, I used `pareto_front` and `Pareto solutions` in the key name of the result dictionary and a log message. I'm not sure if it is user-friendly or not. I guess many users are familiar with the word `best solutions`. Do you think I should replace them with `best_solutions` and `Best solutions`?

https://github.com/toshihikoyanase/hydra/blob/b98efa709ca5f43314e8d8bbec7d8f52eed01dcb/plugins/hydra_optuna_sweeper/hydra_plugins/hydra_optuna_sweeper/_impl.py#L263-L265

Reply:
Regarding the `Pareto optimal solutions`, I add a link to a Wikipedia article about [Pareto efficiency](https://en.wikipedia.org/wiki/Pareto_efficiency) in the commit https://github.com/facebookresearch/hydra/pull/1294/commits/976a916a0ec601400e9ac1469652eebf83c80f30.

> I guess many users are familiar with the word best solutions or solutions. Do you think I should replace them with best_solutions or solutions?

I replaced `pareto_front` that is a key name in the result yaml with `solutions` in the commit (https://github.com/facebookresearch/hydra/pull/1294/commits/d27267c1739b4a8b20238e3b502e78116a6abe5d).

PTAL.